README.md

```diff
@@ -1,17 +1,82 @@
-[OpenVINO™](https://github.com/openvinotoolkit/openvino) Intermediate Representation (IR) models.
+[**OpenVINO™**](https://github.com/openvinotoolkit/openvino) Intermediate Representation (IR) models.
+
+Given **OpenVINO** models and input images, **OpenVINO XAI** generates **saliency maps**
+which highlights regions of the interest in the inputs from the models' perspective
+to help users understand the reason why the complex AI models output such responses.
+
+---
+
+## Features
+
+### What's new in v1.0.0
+
+* Support generation of classification and detection per-class and per-image saliency maps
+* Enable White-Box ([ReciproCAM](https://arxiv.org/abs/2209.14074)) and Black-Box ([RISE](https://arxiv.org/abs/1806.07421v3)) eXplainable AI algorithms
+* Support CNNs and Transformer-based architectures (validation on diverse set of [timm](https://github.com/huggingface/pytorch-image-models) models)
+* Enable `Explainer` (stateful object) as the main interface for XAI algorithms
+* Support `AUTO` mode by default to detect the best XAI method for given models
+* Expose `insert_xai` functional API to support XAI head insertion for OpenVINO IR models
+
+Please refer to the [change logs](CHANGELOG.md) for the full release history.
 
-## Documentation
+### Supported XAI methods
 
-OpenVINO XAI API documentation can be found [here](https://openvinotoolkit.github.io/openvino_xai/).
+At the moment, *Image Classification* and *Object Detection* tasks are supported for the *Computer Vision* domain.
+*Black-Box* (model agnostic but slow) methods and *White-Box* (model specific but fast) methods are supported:
+
+| Domain          | Task                 | Type      | Algorithm           | Links |
+|-----------------|----------------------|-----------|---------------------|-------|
+| Comptuer Vision | Image Classification | Black-Box | RISE                | [arxiv](https://arxiv.org/abs/1806.07421v3) / [src](openvino_xai/methods/black_box/rise.py) |
+|                 |                      | White-Box | ReciproCAM          | [arxiv](https://arxiv.org/abs/2209.14074) / [src](openvino_xai/methods/white_box/recipro_cam.py) |
+|                 |                      |           | VITReciproCAM       | [arxiv](https://arxiv.org/abs/2310.02588) / [src](openvino_xai/methods/white_box/recipro_cam.py) |
+|                 |                      |           | ActivationMap       | experimental / [src](openvino_xai/methods/white_box/activation_map.py) |
+|                 | Object Detection     |           | ClassProbabilityMap | experimental / [src](openvino_xai/methods/white_box/det_class_probability_map.py) |
+
+### Supported explainable models
+
+Most of CNNs and Transformer models from [Pytorch Image Models (timm)](https://github.com/huggingface/pytorch-image-models) are supported and validated.
+
+Please refer to the following known issues for unsupported models.
```

Comment:
This might be the main issue, let's put it on the first place
https://github.com/openvinotoolkit/openvino_xai/issues/26

Reply:
https://github.com/openvinotoolkit/openvino_xai/pull/34/commits/dd5776f6a040fd36580f81175d4702cae4a78943